lib/hammer_cli_foreman/compute_resource/gce.rb

```diff
@@ -10,13 +10,17 @@ def compute_attributes
             'machine_type',
             'image_id',
             'network',
-            'external_ip'
+            'associate_external_ip'
         ]
       end
 
       def interfaces_attrs_name
         "network_interfaces_nics_attributes"
       end
+
+      def mandatory_resource_options
```

Comment:
where do you use this function?

Reply:
`compute_resource create` command is using this [method](https://github.com/theforeman/hammer-cli-foreman/blob/master/lib/hammer_cli_foreman/compute_resource.rb#L94).